MessengerOnFileForLinux/Common/test/TimeManagmentTest.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+#include <ctime>
+#include <regex>
+#include <iostream>
+
+#include "TimeManagment.hpp"
+
+TEST(ConvertTimeToString, TimeFormatSizeIsNineteen)
+{
+    const auto timeAsString = TimeManagment::convertTimeToString(TimeManagment::getActualDate());
+    constexpr auto expectedTimeSize = 19;
+    EXPECT_EQ(timeAsString.size(), expectedTimeSize);
+}
+
+TEST(ConvertTimeToString, TimeFormatIsCorrect)
+{
+    const auto unixEpochTimeAsString = TimeManagment::convertTimeToString(std::time_t{});
+    std::regex expectedTimeFormat("([0-9]{4})-([0-9]{2})-([0-9]{2}) ([0-9]{2}):([0-9]{2}):([0-9]{2})");
```

Comment:
I think we should to search for better solution then using regex (?)

Reply:
What is wrong with regex?